holopy/core/io/vis.py

```diff
@@ -341,7 +341,7 @@ def show_scatterer_slices(scatterer, spacing):
         voxel spacing for the visualization
     """
     vol = scatterer.voxelate(spacing, 0)
-    show2d(vol)
+    Show2D(vol)
```

Comment:
You should try calling `hp.core.io.vis.display_image` before passing to `Show2D` That function is meant to massage any type of input to a standardized xarray output for visualization or saving. Hopefully that removes the need to check if im has a `voxel` attribute later on. `display_image` is a bit messy, so feel free to ask me if you aren't sure of anything.